### PR TITLE
Bug 1776503 - [SELF-SERVICE] New form to add new version and milestone to select products when a new Firefox release comes out

### PR DESCRIPTION
**Pull request by dklawren**
This new form will allow for quickly adding a new Firefox related milestone/version to the products selected. More details are in the bug.

To test:

```bash
$ docker compose down && docker compose build && docker compose up
```

Set your browsers proxy to localhost:1080. Go to http://bmo.test and login as the admin@mozilla.bugs user. Password is in the README.  You can then add a previous milestone and version to the Firefox product of 'Firefox 100' and '100 Branch' respectively. Then access the new release form from the Administrative menu. You should be able to select Firefox from both selects and add 101 to the test fields. When looking at the versions and milestones list, you should see the new 'Firefox 101' and '101 Branch' values create.

Ping me for any questions.

### CODE REVIEW

**Comment by dklawren** [comment]
Need a few more changes before ready for review.

**Review by cgsheeh — CHANGES_REQUESTED** [review]

File: Bugzilla/App/BMO/NewRelease.pm [hunk 1]
```diff
@@ -179,4 +166,18 @@ sub _add_value {
   }
 }
 
+# Helper function to return a fully formatted version or milestone
+sub _get_formatted_value {
+  my ($product, $value, $type) = @_;
+  use Bugzilla::Logging;
+  DEBUG("product: $product");
```
Comment (cgsheeh) [inline Bugzilla/App/BMO/NewRelease.pm:173]: Development debugging lines? If we do want to log this we should import `Bugzilla::Logging` at the top.

**Review by cgsheeh — CHANGES_REQUESTED** [review]
LGTM except the debug calls.

File: Bugzilla/App/BMO/NewRelease.pm (hunk 1 above)
Comment (dklawren) [reply Bugzilla/App/BMO/NewRelease.pm]: Ugh. Totally left that there by mistake. Pushing change to remove.

**Review by cgsheeh — APPROVED** [review]

**Pull request merged**